src/main/java/com/adobe/epubcheck/messages/MessageDictionary.java

```diff
@@ -282,6 +282,8 @@ public Message getMessage(MessageId id)
       map.put(MessageId.OPF_082, Severity.ERROR);
       map.put(MessageId.OPF_083, Severity.ERROR);
       map.put(MessageId.OPF_084, Severity.ERROR);
+      map.put(MessageId.OPF_085, Severity.WARNING);
```

Comment:
I don't think EpubCheck can issue a WARNING for language tag variants, as there are legitimate uses for the variant tag. See for instance [W3C’s explainer on the language tags](https://www.w3.org/International/articles/language-tags/#variants).

At best this can be a HINT, but then I'm not quite sure it's worth implementing if it's just informative?